src/contracts/caller.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * a contract calling quadratic equation contract
+ */
+
+import {
+    Addr,
+    ByteString,
+    SmartContract,
+    Utils,
+    assert,
+    byteString2Int,
+    hash256,
+    len,
+    method,
+    prop,
+    slice,
+} from 'scrypt-ts'
+import { Coeff } from './callee'
+import { TxUtil } from './txUtil'
+
+

```

Comment:
Always reference the article if there's any https://xiaohuiliu.medium.com/inter-contract-call-on-bitcoin-f51869c08be

Reply:
Ok sir